### ISSUE TITLE
Demo Settings Sidebar: Retrieve and display list of available retailers for new retail manager

### ISSUE THREAD

**Issue by jakepeyser** [IBM-Bluemix/logistics-wizard#137]
Front end changes to use API from #97 to populate list of potential locations for retail manager

Story #11

**Comment by jakepeyser** [comment]
@Austinauth same goes here...is this invalid based on new design?

**Comment by Austinauth** [comment]
@jakepeyser Based off the flow we discussed yesterday (https://github.com/IBM-Bluemix/logistics-wizard/issues/148) we determined:

* A new user would come into the product as a Supply Chain Manager by default. 
* We also determined a Retail Manager would be created by default so the user could switch roles to see the demo from an alternate perspective via the Demo Settings Panel. (Per @colbycheeze's suggestion we use the users location data to assign this retail manager a city).
* Lastly, we determined a user should be able to create a new Retail Manager via the Demo Settings Panel if they so desire.

If we all agree on the above, there is no need to retrieve and display a list of retail locations for the Session Creation screen. However, as we dive into the design of the Demo Settings Panel we *may* want to show a list of Retail Locations to someone who wants to add an additional Retail Manager outside of the default one that is created.
